rules-emerging-threats/2023/Malware/Ursnif/proc_creation_win_malware_ursnif_cmd_redirection.yml

```diff
@@ -0,0 +1,26 @@
+title: Ursnif Redirection Of Discovery Commands
+id: 7aaa5739-12fc-41aa-b98b-23ec27d42bdf
+status: experimental
+description: |
+    Detects the redirection of Ursnif discovery commands as part of the initial execution of the malware.
+references:
+    - Internal Research
```

Comment:
I put `Internal Research`. But in case you have a reference, that would be better.